package.json

```diff
@@ -48,7 +48,7 @@
     "themes-generate": "esbuild --target=es6 ./scripts/cli/generateSassVariableFiles.ts --bundle --platform=node --tsconfig=./scripts/cli/tsconfig.json | node",
     "themes:usage": "eslint . --ignore-pattern '*.test.ts*' --ignore-pattern '*.spec.ts*' --cache --plugin '@grafana' --rule '{ @grafana/theme-token-usage: \"error\" }'",
     "typecheck": "tsc --noEmit && yarn run packages:typecheck",
-    "plugins:build-bundled": "find plugins-bundled -name package.json -not -path '*/node_modules/*' -execdir yarn build \\;",
+    "plugins:build-bundled": "",
```

Comment:
This can't be fully removed yet due to https://github.com/grafana/grafana-build/blob/842fd595ef1f2671bc4443b754adc215822887e6/frontend/build.go#L19